etstool.py

```diff
@@ -423,6 +423,7 @@ def build_changelog(ctx):
 # Utility routines
 # ----------------------------------------------------------------------------
 
+
```

Comment:
Uh! Somehow Travis CI did not run on #246 and Appveyor job does not check flake8. Sorry about that.

Reply:
Ah I see, no worries!